cmd/trust/sudi.go

```diff
@@ -11,8 +11,56 @@ import (
 
 	"github.com/apex/log"
 	"github.com/google/uuid"
+	"github.com/urfave/cli"
 )
 
+var genSudiCmd = cli.Command{
+	Name:      "gen-sudi",
+	Usage:     "Generate and sign sudi cert",
+	UsageText: "CACert, private-key, output-dir, product-uuid [, machine-uuid]",
```

Comment:
Just wondering... why not have the user enter a keysetname to acquire the CACert and privkey? The other subcommands ask for a keysetname.

Reply:
Well, I think @smoser  is saying he's using this in tests without using the full 'project-machine' XDG paths at all.

I'm not sure which way we want to go there.  I dont' think expecting tests to run a trust command to set those up is too much to ask, if they're going to use trust to then generate a SUDI key.

Reply:
@hallyn  is right.  I would want to generate a sudi key without using 'mos'.

I had commented about that in the git commit message [here](https://github.com/project-machine/trust/pull/20/commits/5094a854d9234e48bbda49b57c631a6cf0eca3ad):

> This admittedly doesn't fit with the other trust sub-commands too well,
> since the others are all expecting 'mos' tree paths.  That was
> problematic for me as I just wanted to generate a cert.pem and
> privkey.pem.
> 
> But it felt like maybe trust is too inter-twined with mos if it is only
> able to read paths that are written by mos.  Should those functions
> that expect/enforce directory structure be part of the 'mos' binary and
> calling exposed api calls from trust?

I'm fine to drop this "top" commit that adds the sub-command for now.

Reply:
I'm fine taking it, too ("for now").  Let me know what you prefer.